src/modules/auth/auth.module.ts

```diff
@@ -68,6 +71,16 @@ export class AuthModule implements NestModule {
         { path: '/v1/login/siwe/initiate', method: RequestMethod.POST },
         { path: '/v1/login/siwe/verify', method: RequestMethod.POST }
       )
-      .forRoutes({ path: '/*', method: RequestMethod.ALL });
+      .forRoutes({ path: '/*', method: RequestMethod.ALL })
```

Comment:
Would it be better just to have `POST` here as it is only the `POST` endpoint that we are trying to disable (not that there is a `GET` at this point)?
```suggestion
      .forRoutes({ path: '/*', method: RequestMethod.POST })
```